frontend/src/ui/Access.tsx

```diff
@@ -704,6 +920,7 @@ const insertBuiltinRoleInfo = (
     acl: Acl,
     knownGroups: AccessKnownRolesData$data["knownGroups"][number][],
     i18n: i18n,
+    permissionLevel: PermissionLevel,
```

Comment:
I would rather add a `addAnonymous: bool` parameter here. Otherwise it's fairly random that the caller passes `.highest` and that it is only compared to `admin` here.